src/components/error-summary/index.md.njk

```diff
@@ -21,7 +21,14 @@ Always show an error summary when there is a validation error, even if there’s
 
 ## How it works
 
-As well as showing an error summary, follow the [validation pattern](/patterns/validation) - for example, by adding ‘Error: ’ to the beginning of the page `<title>`.
+You must:
+
++ move keyboard focus to the error summary
++ include the heading ‘There is a problem’
++ link to each of the answers that have validation errors
++ make sure the error messages in the error summary are worded the same as those which appear next to the inputs with errors
```

Comment:
In most cases I reckon this is correct - but we had cases when working on Passports where it made sense for the error next to the field to be different from the error in the summary list. I'll try to find an example.